src/lib/y2storage/callbacks/activate_callbacks.rb

```diff
@@ -21,18 +21,27 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
 require "storage"
+require "y2storage/dialogs/callbacks/activate_luks"
 
 module Y2Storage
-  # class to implement callbacks used during activate in ruby
+  # class to implement callbacks used during activate
   class ActivateCallbacks < Storage::ActivateCallbacks
+    include Yast::Logger
```

Comment:
Sure, I was not pointing fingers or nothing like that. Don't take it personally. 😄 As I said, "I just realized", which implies somebody (me in this case, but no matter who) overlooked it in the previous PR.

Shit happens. :wink: That's (partially) why we use code review.